src/views/sessions/SessionDetail/SessionCommands.vue

```diff
@@ -58,10 +58,13 @@ export default {
         hasRightActions: false
       }
     }
-  }
+  },
+  async mounted() {
+    const drawActionMeta = await this.$store.dispatch('common/getDrawerActionMeta')
 
+    if (drawActionMeta && drawActionMeta.id) {
+      this.tableConfig.url = `/api/v1/terminal/commands/?session_id=${drawActionMeta.id}`
+    }
+  }
 }
 </script>
-
-<style scoped>
-</style>
```

Comment:
The main difference between the two versions of this code snippet is that it uses async functions and promises to handle asynchronous requests, which makes the code more concise and readable.

In terms of potential issues/optimization suggestions, using an alias instead of a direct reference (`import` vs `export`) can help maintain readability and avoid repetition.

This code appears to be syntactically correct and does not contain any bugs or major deviations from expected functionality. No further modifications seem necessary at this point in time.